airdrop-cli/src/commands/tally.ts

```diff
@@ -6,8 +6,7 @@ import { invoke } from "../invoke";
 
 @command({
   brief: "Tally UBQ airdrop.",
-  description:
-    "Includes all repositories with payments on dev branch since 01-01-2023.",
+  description: "Includes all repositories with payments on dev branch since 01-01-2023.",
```

Comment:
What does "on dev branch" mean? Issues are not associated with branches.

Reply:
well it's actually the repo default branch which is typically development for UBQ repos but it's whatever the current default branch